### DISCUSSION TITLE
Bulk import existing files from cloud storage.

### DISCUSSION THREAD

**Discussion by thedanielfactor** [payloadcms/payload#1041] [Feature Requests & Ideas]
### Bulk import existing files from cloud storage.

As a cloud storage user, I need to be able to point at an existing folder in the cloud storage and import all images (future -> documents) in that path to a collection. This will allow me to get up to speed with payload faster as I already have all of my images in the CMS to use.

It could be that it will execute on initial creation of the collection only.  Or part the cloud storage plugin exclusively.

**Comment by jmikrut** [comment]
Hey @thedanielfactor — this should be totally doable! You will have to write a script to fetch all images from your cloud storage, then find the matching document that already exists in your CMS, and then update the matched document with Payload's Local API where you pass in the matched file to the `update` operation. 

I don't think anyone has done this, but it can certainly be done!
